tools/package.json

```diff
@@ -28,6 +29,7 @@
     "build-moonbeam-launch": "yarn add PureStake/polkadot-launch#moonbeam-launch",
     "moonbeam-launch": "yarn run build-moonbeam-launch && mkdir -p specFiles && ts-node moonbeam-launch.ts",
     "moonbeam-test": "yarn run build-moonbeam-launch && mkdir -p specFiles && ts-node moonbeam-test.ts",
-    "test-staking": "yarn run build-moonbeam-launch && mkdir -p specFiles && ts-node test-staking.ts"
+    "test-staking": "yarn run build-moonbeam-launch && mkdir -p specFiles && ts-node test-staking.ts",
+    "launch": "ts-node launch"
```

Comment:
WOuld be nice to see an example of a cli call somewhere with the arguments

Reply:
I added it in the README